src/SupportedEntities.php

```diff
@@ -0,0 +1,1459 @@
+<?php
+
+namespace Drupal\civicrm_entity;
+
+/**
+ * Defines supported entities.
+ *
+ * See `$civicrm_entity_info['civicrm_event']` for implemented usage. This might
+ * not be required. Perhaps we just implement entity class definitions using
+ * this information. Providing entity type derivatives seems hard, and possibly
+ * overhead when we just can provide the classes directly.
```

Comment:
Hm, ok, I guess we do need the annotation or derivatives? It'd be best not to have to make a class with an annotation because we're talking about A LOT of classes (dozens?). If that needs derivatives, I think that's what we should do.